public/app/features/dashboard/components/PanelEditor/PanelOptionsTab.tsx

```diff
@@ -26,6 +50,9 @@ export const PanelOptionsTab: FC<Props> = ({
   onPanelConfigChange,
   onPanelOptionsChanged,
 }) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
```

Comment:
You can use `useStyles` instead when styles getter depends on  theme only (https://github.com/grafana/grafana/blob/master/contribute/style-guides/themes.md#usestyles-hook)